src/client.coffee

```diff
@@ -221,7 +221,7 @@ class Client extends EventEmitter
                 @emit 'new_user', message
             else
                 # Check for `pong` response
-                if message.data?.text? and message.data.text == "pong"
+                if message.data.text? and message.data.text == "pong"
```

Comment:
@Vhex this is not a typo but an extra check to prevent crashes in case the message also does not have data object